README.md

```diff
@@ -59,4 +63,4 @@ To deploy the backend to Heroku:
 local postgres database has data and works when you run the app locally,
 then run `heroku pg:push LOCAL_DB_NAME postgres`
 
-- Visit the [application](http://text-thresher.herokuapp.com/api/) to make sure it worked.
\ No newline at end of file
+- Visit the [application](http://text-thresher.herokuapp.com/api/) to make sure it worked.
```

Comment:
i'm curious why git is showing a diff here. the whole `deploy` section should be rewritten anyhow.